src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalTool.java

```diff
@@ -276,63 +276,6 @@ public JsonObjectBuilder toJson() {
         return jab;
     }
 
-    public enum ReservedWord {
```

Comment:
Why are we moving ReservedWord around? What does it have to do with remote stores? (I'll know by the end of my initial scan of the code, I hope.)

Reply:
It isn't directly. Since the URL signing is getting re-used in OpenDP, I was trying to keep in sync with what I gave Bob to avoid any inconsistencies wr.t. to the URlSigner/URLToken classes. It could probably be removed from this PR if someone can show me how to do that without adding a commit that will remove it from Bob's branch when he merges with develop after this PR.

Reply:
My concern has to do with external tools. How much risk is there that we are breaking something in this PR? How much QA should we do? If we keep this change, should the "how to test" for this PR be updated to explain what to test?

With regard to git stuff, we could perhaps ask @adaybujeda for advice. His PRs often have a single commit. I assume he's squashing his commits (something I haven't played with) and rebasing. My dumb way of doing this (getting down to a single commit) is to create a new branch.

Let's first decide if we want the change in or out. Do we want to mess with the external tools code? Then let's decide what if anything to do in git for this branch/PR.

Reply:
@qqmyers and I decide to leave the external tools change in (it's just an automated refactoring, he said). He updated "how to test" to explain the external tools need to be regression tested.

The debate about rebasing continues in #5379 😄 ... but I'm resolving this thread.